controllers/release/release_adapter_test.go

```diff
@@ -236,6 +236,17 @@ var _ = Describe("Release Adapter", Ordered, func() {
 		}, time.Second*10).Should(BeTrue())
 	})
 
+	It("can finalize a Release", func() {
+		// Release should be properly finalized if there is no ReleasePipelineRun
+		Expect(adapter.finalizeRelease()).To(Succeed())
+
+		// It should finalize properly as well when there's a ReleasePipelineRun
+		result, err := adapter.EnsureReleasePipelineRunExists()
+		Expect(result.CancelRequest).To(BeFalse())
+		Expect(err).NotTo(HaveOccurred())
+		Expect(adapter.finalizeRelease()).To(Succeed())
```

Comment:
maybe adding an `Eventually` block with Get here to make sure it was really gone?

Reply:
Absolutely right. My bad.